modules/dind/dind.go

```diff
@@ -0,0 +1,118 @@
+package dind
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/api/types/mount"
+
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+var (
+	// containerPorts {
+	defaultDockerDaemonPort = "2375/tcp"
+)
+
+// DinDContainer represents the Docker in Docker container type used in the module
+type DinDContainer struct {
+	testcontainers.Container
+}
+
+// Deprecated: use Run instead
```

Comment:
suggestion: let's remove this deprecated block. It's probably coming from our template, so I'll remove it on our side. In the meantime, please remove it here, and from [here](https://github.com/testcontainers/testcontainers-go/pull/3004/files#diff-c459e38db94b50292992f14e240100424088353eaa2738a7f0e624bd429f5893R29-R30)